tensornetwork/tests/network_test.py

```diff
@@ -78,6 +78,20 @@ def test_tnwork_copy_identities(backend):
     assert not edge_dict[edge] is edge
 
 
+def test_tnwork_copy_subgraph(backend):
+  a = tn.Node(np.random.rand(3, 3, 3), name='a', backend=backend)
+  b = tn.Node(np.random.rand(3, 3, 3), name='b', backend=backend)
+  c = tn.Node(np.random.rand(3, 3, 3), name='c', backend=backend)
+  a[0] ^ b[1]
+  edge2 = b[2] ^ c[0]
```

Comment:
If you add another test where `edge2 = c[0] ^ b[2]` does it still work?

Reply:
added a added a test for it and works now.